users/vieting/experiments/switchboard/ctc/feat/conformer_baseline.py

```diff
@@ -154,6 +155,25 @@ def run_test_mel():
                 },
                 report_args={"architecture": "conf-wei", "lr": "wei_peak_4e-4", "specaug": "wei_adapt_80dim"},
             ),
+            "lgm80_conf-audio_perturbation": dict(
+                returnn_args={
+                    "conformer_type": "wei",
+                    "extra_args": {
+                        "audio_perturb_args": {
+                            "speed": {"prob": 0.6, "minimum": 0.88, "maximum": 1.12},
+                            "tempo": {"prob": 0.6, "minimum": 0.83, "maximum": 1.17},
+                        },
+                        "audio_perturb_runner": CodeWrapper("WaveformPerturbation(**audio_perturb_args)")
+                    },
+                    **returnn_args
+                },
+                feature_args=feature_args,
+                lr_args={
+                    "peak_lr": 4e-4, "start_lr": 1.325e-05, "end_lr": 1e-5,
+                    "increase_epochs": 119, "peak_epochs": 2, "decrease_epochs": 119, "final_epochs": 0,
+                },
+                report_args={"architecture": "conf-wei", "lr": "wei_peak_4e-4"},
```

Comment:
```suggestion
                report_args={"architecture": "conf-wei", "lr": "wei_peak_4e-4", "speed": "0.6_0.88_1.12", "tempo": "0.6_0.83_1.17"},
```
Something like this will add information about the perturbation to the csv report that is created. Please check if the line is too long.